modules/era/sql/era_droplist.sql

```diff
@@ -22346,10 +22320,10 @@ INSERT INTO `mob_droplist` VALUES (2697,2,0,1000,751,0);    -- Platinum Beastcoi
 
 -- ZoneID: 138 - Yagudo Chanter
 -- ZoneID: 151 - Yagudo Chanter
-INSERT INTO `mob_droplist` VALUES (2698,0,0,1000,4980,@UNCOMMON); -- Scroll Of Foe Requiem V (Uncommon, 10%)
-INSERT INTO `mob_droplist` VALUES (2698,0,0,1000,5020,@UNCOMMON); -- Scroll Of Gold Capriccio (Uncommon, 10%)
-INSERT INTO `mob_droplist` VALUES (2698,0,0,1000,5030,@RARE);     -- Scroll Of Carnage Elegy (Rare, 5%)
-INSERT INTO `mob_droplist` VALUES (2698,2,0,1000,749,0);          -- Mythril Beastcoin (Steal)
+INSERT INTO `mob_droplist` VALUES (2698,0,0,1000,5020,@RARE);  -- Scroll Of Gold Capriccio (Rare, 5%)
+INSERT INTO `mob_droplist` VALUES (2698,0,0,1000,4980,@VRARE); -- Scroll Of Foe Requiem V (Very Rare, 1%)
```

Comment:
http://ffxidb.com/zones/151/yagudo-chanter

With this many data points it is more likely to be a 5% drop rate. Generally if you put things at 1% the TH4 drop rate is only 2.5%. Most likely with these low drop rates over a large data pool you should see a lower drop rate than the actual one making a lot of these 5% instead of 1% (makes sense in my head)